### ISSUE TITLE
Bug of restarting training?

### ISSUE THREAD

**Issue by kxhit** [eloialonso/diamond#24]
Thanks for this great work!

I just launched the training under the csgo branch and it seems the learning_rate is reset after the first epoch, the loss also.

Could you let me know if this is on purpose and why, or it is a small bug? Thanks!

<img width="311" alt="1729542146376" src="https://github.com/user-attachments/assets/71b7e673-ade6-459e-af5e-3450931c55db">

**Comment by kxhit** [comment]
Sorry, my bad. It was a wandb logging debug problem I forgot to roll back... Cheers!
